### PR TITLE
Treat SQLite DB and mostly binaries as binaries #43  

### LINKED ISSUE #43: A sqlite DB is detected as not a binary
This file https://github.com/linq2db/linq2db/raw/5e26fa1693208b92b0791dff445ab7b42e79fb1b/Data/Northwind.sqlite is not detected correctly as binary 

```
>>> is_binary('Northwind.sqlite')
DEBUG:binaryornot.helpers:nontext_ratio1: 0.9697265625
DEBUG:binaryornot.helpers:nontext_ratio2: 0.98828125
DEBUG:binaryornot.helpers:is_likely_binary: True
DEBUG:binaryornot.helpers:detected_encoding: {'confidence': 0.9377949365075317, 'language': 'Bulgarian', 'encoding': 'windows-1251'}
DEBUG:binaryornot.helpers:success: decodable_as_unicode: True
DEBUG:binaryornot.helpers:failure: decodable_as_unicode: True
False
```

IMHO it should be a binary

### PR DESCRIPTION
**Pull request by pombredanne**
This PR fixes #43 

Signed-off-by: Philippe Ombredanne <pombredanne@nexb.com>

### CODE REVIEW

**Comment by pombredanne** [comment]
@audreyr BTW do you need help with the maintenance of this project?

**Comment by pombredanne** [comment]
Let me revert the Appevyor config changes. :|

**Comment by pydanny** [comment]
@pombredanne Thank you for this effort so far! 

Work has given me the chance to support this and other Cookiecutter-related projects going forward. Any help you give toward maintaining the project will be greatly appreciated.

File: binaryornot/helpers.py [hunk 1]
```diff
@@ -84,6 +84,9 @@ def is_binary_string(bytes_to_check):
     nontext_ratio2 = float(len(high_chars)) / float(len(bytes_to_check))
     logger.debug('nontext_ratio2: %(nontext_ratio2)r', locals())
 
+    if nontext_ratio1 > 0.90 and nontext_ratio2 > 0.90:
```
Comment (pombredanne) [inline binaryornot/helpers.py]: The rationale is that if this is mostly non-ascii  (e.g low and high bytes) then this is a binary most likely.

**Comment by pombredanne** [comment]
@pydanny re:

> Work has given me the chance to support this and other Cookiecutter-related projects going forward.  Any help you give toward maintaining the project will be greatly appreciated.

Sure thing! binaryornot is used in scancode-toolkit which is a somewhat popular open source license and copyright scanner so it sees a lot of files. There are some weirdness I noticed recently and I will likely come with a few more tweaks

**Comment by pombredanne** [comment]
@pydanny do you mind to tag and push a release to Pypi?

**Pull request merged**